gel-tokio/src/builder.rs

```diff
@@ -171,17 +172,16 @@ impl Config {
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Clone, Debug)]
 pub(crate) struct ConfigInner {
-    pub address: Address,
+    pub address: gel_stream::Target,
```

Comment:
Let's remember that this is a breaking change.

Reply:
I believe this Address is an internal struct that isn't exposed outside of unstable. I may be wrong -- I can partially revert this if needed.